Library/src/main/java/info/metadude/android/typedpreferences/BasePreference.java

```diff
@@ -0,0 +1,33 @@
+package info.metadude.android.typedpreferences;
+
+import android.content.SharedPreferences;
+
+/**
+ * A wrapper class for a boolean preference.
+ */
+class BasePreference {
```

Comment:
no this should not be public as it should not be use outside this package - also not abstract as it has no abstract methods